db/migrate/20200428081551_add_end_deadline_to_tasks.rb

```diff
@@ -0,0 +1,5 @@
+class AddEndDeadlineToTasks < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tasks, :end_deadline, :datetime, null: false, default: -> { 'NOW()'}
```

Comment:
`end_`つけなくても良いかと思いましたが、こちらいかがでしょうか。
```suggestion
    add_column :tasks, :deadline, :datetime, null: false, default: -> { 'NOW()'}
```

Reply:
「開始までの期限」を入れる可能性を残しておきたく、「end」をつけていました。
こちらはそのままです。